<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Organizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>580</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="viewButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 195, 79);
border-bottom-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(247, 247, 247);
border-left-color: rgb(0, 0, 0);
border: 1px solid #000000;
border-radius: 5;
font: 12pt &quot;Roboto&quot;;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>View Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dashboardButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 164, 197);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(247, 247, 247);
border-left-color: rgb(0, 0, 0);
border: 1px solid #000000;
border-radius: 5;
font: 12pt &quot;Roboto&quot;;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>GDrive Upload</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>-80</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/resources/dino-new.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>160</y>
       <width>685</width>
       <height>70</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Thank you, all pics sorted!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Please check if all looks fine:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #e9acbc;
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(247, 247, 247);
border-left-color: rgb(0, 0, 0);
border: 1px solid #000000;
border-radius: 5;
font: 12pt &quot;Roboto&quot;;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Upload Next</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
